docs/development/extensions-contrib/prometheus.md

```diff
@@ -44,7 +44,7 @@ All the configuration parameters for the Prometheus emitter are under `druid.emi
 | `druid.emitter.prometheus.addHostAsLabel`     | Flag to include the hostname as a prometheus label.                                                                                                                                                                                    | no        | false                                |
 | `druid.emitter.prometheus.addServiceAsLabel`  | Flag to include the druid service name (e.g. `druid/broker`, `druid/coordinator`, etc.) as a prometheus label.                                                                                                                         | no        | false                                |
 | `druid.emitter.prometheus.pushGatewayAddress` | Pushgateway address. Required if using `pushgateway` strategy.                                                                                                                                                                         | no        | none                                 |
-| `druid.emitter.prometheus.flushPeriod`        | Emit metrics to Pushgateway every `flushPeriod` seconds. Required if `pushgateway` strategy is used.                                                                                                                                   | no        | 15                                   |
+| `druid.emitter.prometheus.flushPeriod`        | If strategy is `pushgateway`, emits metrics every `flushPeriod` seconds. Required if `pushgateway` strategy is used. If strategy is `exporter`, configures the metric ttl such that if the metric value is not updated within the `flushPeriod` then it will stop being emitted. Optional if `exporter` strategy is used.                                                                                                                                   | no        | 15 seconds for `pushgateway` strategy. None for `exporter` strategy                                  |
```

Comment:
```suggestion
| `druid.emitter.prometheus.flushPeriod`        | If strategy is `pushgateway`, emits metrics every `flushPeriod` seconds. If strategy is `exporter`, configures the metric ttl such that if the metric value is not updated within the `flushPeriod` then it will stop being emitted.                                                                                                                                   | Required if `pushgateway` strategy is used, optional otherwise.        | 15 seconds for `pushgateway` strategy. None for `exporter` strategy. |
```